test/e2e/upgrade_ironic_test.go

```diff
@@ -28,16 +29,11 @@ func upgradeIronic() {
 	By("Upgrading ironic image based containers")
 	deploy, err := getIronicDeployment(clientSet, ironicDeployName, ironicNamespace)
 	Expect(err).To(BeNil())
-	for i, container := range deploy.Spec.Template.Spec.Containers {
-		switch container.Name {
-		case
-			"mariadb",
-			"ironic-api",
-			"ironic-dnsmasq",
-			"ironic-conductor",
-			"ironic-log-watch",
-			"ironic-inspector":
-			deploy.Spec.Template.Spec.Containers[i].Image = containerRegistry + "/metal3-io/ironic:" + ironicImageTag
+	for _, container := range deploy.Spec.Template.Spec.Containers {
+		if container.Name == "mariadb" {
+			container.Image = containerRegistry + "/metal3-io/mariadb:" + ironicImageTag
```

Comment:
Probably better to create a specific var for mariadb just like 
```suggestion
                        mariadbImageTag := ironicImageTag // or in e2eConfig
			container.Image = containerRegistry + "/metal3-io/mariadb:" + mariadbImageTag
```